ante/fee.go

```diff
@@ -93,3 +162,54 @@ func GetTxPriority(fee sdk.Coins) int64 {
 
 	return priority
 }
+
+// overwrite DenomsSubsetOf from sdk, to allow zero amt coins. e.g. 1stake is DenomsSubsetOf 0stake.
+// DenomsSubsetOf returns true if coins's denoms is subset of coinsB's denoms.
+// if coins is empty set, empty set is any sets' subset
+func DenomsSubsetOf(coins, coinsB sdk.Coins) bool {
```

Comment:
May be good to distinguish the behavior in the function name
```suggestion
func DenomsSubsetOfIncludingZero(coins, coinsB sdk.Coins) bool {
```